cinn/frontend/op_mapper_registry.h

```diff
@@ -47,60 +46,40 @@ class OpMapperContext {
         var_map_(var_map),
         var_model_to_program_map_(var_model_to_program_map) {}
 
-  hlir::framework::Scope* scope_{nullptr};
-  const common::Target& target_;
-  NetBuilder* builder_{nullptr};
+  const auto& Scope() const { return scope_; }
```

Comment:
谷歌编码规范这里建议小写，https://zh-google-styleguide.readthedocs.io/en/latest/google-cpp-styleguide/naming/，但不强制。
```suggestion
  const auto& scope() const { return scope_; }
```